lambda/custom/index.js

```diff
@@ -15,9 +15,31 @@ const APL_DOC = require ('./document/renderPage.json' ) ;
 const TWO_PAGER_COMMANDS =  require ('./document/twoSpeakItemsCommand.json' ) ;
 const ONE_PAGER_COMMANDS =  require ('./document/oneSpeakItemCommand.json' ) ;
 const TOKEN_ID = 'pagerSample';
-let MAIN_DATA_SOURCE = require ('./document/dataSource.json' );
-
-
+let firstTransformer = [
+      {
+          "inputPath": "phraseSI",
+          "outputName": "phraseAsSpeech",
+          "transformer": "ssmlToSpeech"
+      }
+    ];
+let secondTransformer = [
+      {
+          "inputPath": "phraseSI",
+          "outputName": "nextPhraseAsSpeech",
+          "transformer": "ssmlToSpeech"
+      }
+    ];
+
+function makePage(teaser="",reprompt="",firstProperty="",transformer=[]) {
```

Comment:
Can you make these parameter names more descriptive? For instance, transformer is actually a transformerList, FirstProperty should be more descriptive. Reprompt and teaser are both text/speech? It's not clear to me looking at this.

Reply:
Done